app/models/concerns/host_common.rb

```diff
@@ -30,7 +30,7 @@ module HostCommon
 
     alias_method :all_puppetclasses, :classes
 
-    has_many :lookup_values, :finder_sql => Proc.new { LookupValue.where('lookup_values.match' => lookup_value_match).to_sql }, :dependent => :destroy
+    has_many :lookup_values, :finder_sql => Proc.new { LookupValue.where('lookup_values.match' => lookup_value_match).to_sql }, :dependent => :destroy, :validate => false
```

Comment:
Hmmm... for some reason tests don't indicate that, I remove the validation and lookup values are cloned properly. I'm also concerned about `Host::Managed` lookup_values here.